ui/signed-in/src/commonMain/kotlin/social/androiddev/signedin/composables/SignedInRootContent.kt

```diff
@@ -26,6 +27,7 @@ import social.androiddev.timeline.navigation.TimelineComponent
  * currently signed in.
  * Business logic and decompose navigation is delegated to [SignedInRootComponent].
  */
+@OptIn(ExperimentalDecomposeApi::class)
```

Comment:
Maybe we should move all OptIns into the gradle files

Reply:
Yeah, it was annoying me too. I'll do it as a follow-up and can in line a descriptive comment for what each is for instead of having to duplicate it everywhere I use the annotation.